### PR TITLE
Release 1.0.1 - Minor template update release

### PR DESCRIPTION
**Pull request by a4000**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/readsimulator/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/readsimulator _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/workflows/release-announcements.yml [hunk 1]
```diff
@@ -9,6 +9,11 @@ jobs:
   toot:
     runs-on: ubuntu-latest
     steps:
+      - name: get topics and convert to hashtags
+        id: get_topics
+        run: |
+          curl -s https://nf-co.re/pipelines.json | jq -r '.remote_workflows[] | select(.full_name == "${{ github.repository }}") | .topics[]' | awk '{print "#"$0}' | tr '\n' ' ' >> $GITHUB_OUTPUT
```
Comment (mashehu) [inline .github/workflows/release-announcements.yml:15]:
```suggestion
        run: |
          echo "topics=$(curl -s https://nf-co.re/pipelines.json | jq -r '.remote_workflows[] | select(.full_name == "${{ github.repository }}") | .topics[]' | awk '{print "#"$0}' | tr '\n' ' ')"  >> $GITHUB_OUTPUT
```
A small oversight in the last template release, this is from the current dev branch, to make this action actually work

**Review by mashehu — APPROVED** [review]

File: main.nf [hunk 2]
```diff
@@ -13,68 +13,98 @@ nextflow.enable.dsl = 2
...
-    def citation = '\n' + WorkflowMain.citation(workflow) + '\n'
-    def String command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome GRCh37 -profile docker"
-    log.info logo + paramsHelp(command) + citation + NfcoreTemplate.dashedLine(params.monochrome_logs)
-    System.exit(0)
-}
-
-// Validate input parameters
-if (params.validate_params) {
-    validateParameters()
+// TODO nf-core: Remove this line if you don't need a FASTA file
```
Comment (SPPearce) [inline main.nf:32]: TODO still here

File: CHANGELOG.md
Comment (SPPearce) [inline CHANGELOG.md:1]: Should note that multiqc version has changed 1.19 -> 1.21

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,20 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0.0 - 2024-02-07
+## 1.0.1 - 2024-03-22
```
Comment (SPPearce) [inline CHANGELOG.md:6]:
```suggestion
## 1.0.1 - 2024-03-23
```

File: modules/local/insilicoseq/generate/main.nf
Comment (SPPearce) [inline modules/local/insilicoseq/generate/main.nf:1]: Is there a plan to put this module into the main module repo?

**Review by SPPearce — APPROVED** [review]
Looks good to me, just a few minor things.

File: modules/local/insilicoseq/generate/main.nf
Comment (a4000) [reply modules/local/insilicoseq/generate/main.nf:1]: Yes, I was planning to start adding these modules to the main module repo in the next couple of weeks.

**Pull request merged**